packages/webapp/src/components/EditCropVariety/index.jsx

```diff
@@ -143,12 +141,30 @@ export default function PureEditCropVariety({
 
       <Input
         style={{ marginBottom: '40px' }}
-        label={t('translation:FIELDS.EDIT_FIELD.VARIETY')}
-        type="text"
-        hookFormRegister={varietyRegister}
-        hasLeaf={true}
+        label={t('CROP.VARIETY_COMMON_NAME')}
+        hookFormRegister={commonNameRegister}
+        errors={getInputErrors(errors, FIELD_NAMES.COMMON_NAME)}
```

Comment:
Is the missing `hasLeaf` property here intentional?

Reply:
Yes because when adding a crop there is no leaf, so I assumed that's how it should be for both views.